thunder/tests/test_nvfuser.py

```diff
@@ -850,3 +851,35 @@ def get_num_fusions(cfn):
     assert get_num_fusions(cfn_without_fusion) == 0
 
     nvfuserex.set_fuel(thunder.extend.FUEL_LEVEL.UNLIMITED)
+
+
+@instantiate(
+    dtypes=(thunder.float16, thunder.bfloat16), devicetypes=(devices.DeviceType.CUDA,), executors=(nvFuserExecutor,)
+)
+def test_linear(executor, device: str, dtype: dtypes.dtype):
+
+    def fn(a, b, bias=None):
+        return torch.nn.functional.linear(a, b, bias)
+
+    m, n, k = 128, 64, 32
+    torch_dtype = ltorch.to_torch_dtype(dtype)
+    a = torch.randn((m, k), dtype=torch_dtype, device=device)
+    b = torch.randn((n, k), dtype=torch_dtype, device=device)
```

Comment:
I didn't use it since we only have partial support for linear at the moment, in terms of shape. Will update the tests to use the input generators once the new IR nodes go in.